lib/OfferRepository.js

```diff
@@ -25,7 +27,7 @@ class OfferRepository {
     return new Promise((resolve, reject) => {
       this.OfferContract.new(
         name, offer.origin, offer.category, imageHash,
-        offer.packageWeight, offer.pricePerUnit, marketAddress,
+        offer.packageWeight * 10**WEIGHT_DECIMALS, offer.pricePerUnit * 10**PRICE_DECIMALS, marketAddress,
```

Comment:
We should:
1) Use BigNumber everywhere we use numbers that touch Ethereum
3) Make a nice helper function to make it nice
e.g.:
```
toBigNumberWithDecimals(offer.packageWeight, WEIGHT_DECIMALS)
fromBigNumberWithDecimals(offer.packageWeight, WEIGHT_DECIMALS)
```